crates/services/p2p/src/request_response/messages.rs

```diff
@@ -75,6 +79,7 @@ pub enum NetworkResponse {
 #[derive(Debug, Clone)]
 pub enum OutboundResponse {
     Block(Option<Arc<SealedBlock>>),
+    BlocksInclusive(Vec<SealedBlock>),
```

Comment:
```suggestion
    BlocksInclusive(Option<Arc<Vec<SealedBlock>>),
```

Reply:
Can we close these? Do you still prefer `Option` here? I like `Vec` because we have to check all the values anyway.